test/invoice_item_parser_test.rb

```diff
@@ -4,6 +4,6 @@
 
 class InvoiceItemParserTest < MiniTest::Test
   def test_it_exists
-    assert InvoiceItemParser.new('test/support/sample_invoice_items.csv')
+    assert InvoiceItemParser.new('test/support/invoice_items.csv')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.